File: 프로그래머스/0/120840. 구슬을 나누는 경우의 수/구슬을 나누는 경우의 수.js
function solution(balls, share) {
    var answer = 0;
    answer=combination(balls,share);
    return answer;
}

function combination( n, k){
    let result=1;
    
      for(let i=0;i<k;i++){
            result*=(n-i);
            result/=(i+1);
        }
    return result;
}

File: 프로그래머스/0/120840. 구슬을 나누는 경우의 수/구슬을 나누는 경우의 수.go
func combination(n int, k int) int {
    result := 1
    for i := 0; i < k; i++ {
        result *= (n - i)
        result /= (i + 1)
    }
    return result
}


func solution(balls int, share int) int {
    return combination(balls, share)
}